eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/EventsourcedView.scala

```diff
@@ -415,13 +422,27 @@ trait EventsourcedView extends Actor with Stash {
     }
     case ReplaySuccess(events, progress, iid) => if (iid == instanceId) {
       events.foreach(receiveEvent)
+      // reset retry attempts
+      context.become(initiating(settings.replayRetryMax))
       replay(progress + 1L)
     }
-    case ReplayFailure(cause, iid) => if (iid == instanceId) {
-      logger.error(cause, s"replay failed, stopping self")
-      Try(onRecovery(Failure(cause)))
-      context.stop(self)
+    case ReplayFailure(cause, progress, iid) => if (iid == instanceId) {
+      if (replayAttempts < 1) {
+        // all replay attempts exceeded -> stop the actor
+        logger.error(cause, "replay failed ({} retries reached), stopping self", settings.replayRetryMax)
+        Try(onRecovery(Failure(cause)))
+        context.stop(self)
+      } else {
+        // retry replay request while decreasing the remaining attempts
+        val attemptsRemaining = replayAttempts - 1
+        logger.warning("replay failed - {} attempts remaining [{}] - scheduling retry in {}ms",
```

Comment:
`logger.warning("replay failed: [{}] ({} replay attempts remaining), scheduling retry in {}ms", cause.getMessage, attemptsRemaining, settings.replayRetryDelay.toMillis)` (more consistent with previous `logger.error`)